### PR TITLE
[SPE-585] add cli command to upload cert ref

### PR DESCRIPTION
**Pull request by donaltuohy**
# Why
A user should be able to upload a cert ref from the cli. 

# How
Used the EIFHasher from https://crates.io/crates/aws-nitro-enclaves-image-format to generate the PCR8 hash of the signing cert passed in.
When the PCR8 is generated, the validity of the cert is pulled and validated and then uploaded to the api.

### CODE REVIEW

**Review by hanneary — APPROVED** [review]

File: src/cli/cert.rs [hunk 1]
```diff
@@ -66,6 +82,27 @@ pub fn run(cert_args: CertArgs) -> exitcode::ExitCode {
...
+                            e
+                        );
+                        return e.exitcode();
+                    }
+                };
+            let success_msg = serde_json::json!({
+                "status": "success",
+                "output": cert_ref,
+            });
+            println!("{}", serde_json::to_string(&success_msg).unwrap());
```
Comment (lfarrel6) [inline src/cli/cert.rs]: Should there be a human readable output if stdout isn't being piped here?

File: src/cli/cert.rs [hunk 2]
```diff
@@ -66,6 +82,27 @@ pub fn run(cert_args: CertArgs) -> exitcode::ExitCode {
                 println!("{}", serde_json::to_string(&success_msg).unwrap());
             };
         }
+        CertCommands::Upload(upload_args) => {
+            let api_key = get_api_key!();
```
Comment (lfarrel6) [inline src/cli/cert.rs:93]: Could be nice to default the cert path to the value in the toml if not given

File: src/cli/cert.rs (hunk 1 above)
Comment (donaltuohy) [reply src/cli/cert.rs]: There should, updated 👍

File: src/cli/cert.rs (hunk 2 above)
Comment (donaltuohy) [reply src/cli/cert.rs:93]: Updated as well!

**Review by lfarrel6 — APPROVED** [review]

**Pull request merged**